consumer/consumerdata/consumerdata.go

```diff
@@ -41,13 +41,17 @@ type TraceData struct {
 // OTLPTraceData is a struct that groups proto spans with a resource. This is the
 // newer version of TraceData, using OTLP-based representation.
 type OTLPTraceData struct {
-	ResourceSpanList []*otlptrace.ResourceSpans
+	resourceSpanList []*otlptrace.ResourceSpans
```

Comment:
That's one of the reasons. The other is that I may keep it private and require certain modifications to go through functions. It will become clearer in future PRs. Let's keep it private for now.